### PR TITLE
KFSPTS-4490: Added checkbox on REQS notes to indicate whether they

### PR DESCRIPTION
**Pull request by bhutchinson**
should be copied to the PO. (KFSPTS-5924: put back what once was lost)

### CODE REVIEW

**Comment by bhutchinson** [comment]
@cah292 NOTE after consulting with the Kuali Financials product team, I had to move the NoteExtendedAttribute from krad to sys to make this work with the latest version. We think that's a bug and James will be reporting one issue and I'll be reporting another.

File: src/main/webapp/WEB-INF/tags/module/purap/camsDetail.tag [hunk 1]
```diff
@@ -234,8 +234,7 @@
                 <purap:camsLocation camsLocationAttributes="${camsLocationAttributes}" ctr="${ctr}"
                                     ctr2="new"
                                     camsAssetLocationProperty="${locationPrefix}newPurchasingCapitalAssetLocationLine"
-                                    availability="${availability}" poItemInactive="${poItemInactive}"
-                                    fullEntryMode="${fullEntryMode}"/>
+                                    availability="${availability}" poItemInactive="${poItemInactive}"/>
```
Comment (cah292) [inline src/main/webapp/WEB-INF/tags/module/purap/camsDetail.tag]: I don't recall this change being part of the original PR for this feature on the KFS 5.3.1 line. Was this supposed to be for a different PR?

**Comment by cah292** [comment]
Changes look good overall, but I did have a question about a file change that I didn't think was originally present in the KFS 5.3.1 version of this feature. Will wait for feedback on that.

File: src/main/webapp/WEB-INF/tags/module/purap/camsDetail.tag (hunk 1 above)
Comment (bhutchinson) [inline src/main/webapp/WEB-INF/tags/module/purap/camsDetail.tag]: It was something I had missed when making changes earlier and didn't notice until I tried testing this one by opening a REQS via the webapp. I could separate it out, but it seemed easier to just include the fix here.

Comment (cah292) [inline src/main/webapp/WEB-INF/tags/module/purap/camsDetail.tag]: Ok, sounds good given that this is for the upgrade branch.

**Comment by cah292** [comment]
Changes are good and the extra file change is okay to include as well. Merging.
